lib/index.js

```diff
@@ -172,7 +172,6 @@ internals.parseServer = function (server, relativeTo) {
 
 internals.parsePlugin = function (plugin, relativeTo) {
 
-    plugin = Hoek.cloneWithShallow(plugin, ['options']);
```

Comment:
I see no reason to removing that, maybe add 'plugin.options' to the shallow cloning if that's causing problems.